.travis.yml

```diff
@@ -0,0 +1,89 @@
+dist: trusty
+
+sudo: required
+
+language: java
+
+jdk:
+- openjdk8
+- oraclejdk8
+
+services:
+- docker
+
+notifications:
+  email:
+    recipients:
+    - arfathpasha@gmail.com
+    on_success: never
+    on_failure: always
+
+env:
+  global:
+  - MONGODB_VER=mongodb-linux-x86_64-2.6.12 ANT_TEST=test WIRED_TIGER=false
+  - JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
+  # TEST_TOKEN
+  - secure: tguBGgSgTvVY/ayL0IFA08JM8Nu0ys0iwU2nte2oZIl8J6db+8FwzV6SlXeRjYYJYYHu3uOobYb2i1OpEGfRH2aE8+Q0TZzuupSlKb2cEReV9SAL4w0cb0azg06gjKmY+NoA18lCDZKV7X15Zh9cGkRvcM0Mjbhbbdek3n8vf1h5F1Gp6sV7MHSwQEGRmvmyEy/qTb7D1qIVEyZaxWV+ou3kd4eagiaJr0P+rvqBUOvoqWsUrroR3Vfs2A5J5gy4IeZJhpxwJsJz3UapuWcmYFV+fU/DgGdx3oaG7O5+rBH/v4uzZGaha7xuLFP93UURpxPbQyXKhsE1OiRW+9svpZw3yAS5/PN49GIWcqg3vNECkuyPBO9iSDLzz6iUNiErlsePFKKIlk/qzwOJimmz1+8ZCeDHZXUv81JK+RhKK1dlWECZcpYrr4YsTV4GVkVORIcseIy2jLmvRIer0pLiauOPooiNVcCv+TiYkwfeZdO8QiwsxN1WBzjzniwS3cA6z/AKmm0qJNpu37JTO19AiKEqgcjEfbftSp7S/nnIdGxObFt84iToGb3XfdFVeI8O3B9ngFxuxILb0hYfaXm5TfD2WIUL8pL6g3GVVapFJDkal1PTmc6uq6jU8UW+3SirqRd34ELHcoOKI03cnVdaAgCbKViTdGZtjZjpuBkHz48=
```

Comment:
Please change this to:
  secure: "DC0WOpe4kkbV5QiMkhJLXpR+e1aCFHP0MPI7+adBSsEmaoIZPeN19jykD9DvATxTmlPkyRUilsMfJ+gx+P1Y4pJ0dfWsDXAgUOrf3g7QbPixEfrLBLqmY7oALcP6bKAIlZjgAtG4uFEmYJnZ9qP/5IWnzZsshJzscxxb7LWJok9hAaTs0rD5I6sHfru+/2mJnhcSE6WUhYbL88pm/3MJ0zIVoKMGPukYJY6YH6MzrMe///N5NScF6eqONFA0KVYp7PIHRAcWl82ivWhrN5Hrgq4p05+y8aq8eO8SNm4rbjayNX6Ya4zK8a5phv8zmIv0YCxlcB/e/vcLLMoXzoxZry7a7FwxCO/ao9v4kijNmOfrvp2fqRUl0OY7jQY2kMF7jowMBNoMzqweuTLohmTg2QCr/uaq6h3FxNwxesBRmHFnS4Bs3j6aQiomAuX5n+JunFG576wuFHih09dtu9ScUNr7rCOE5WCYQlsOuYFeigyv7/4iTZDr31OwwqnK48Bm+Nyw1vSp+274CjHlyCaMTSfd17kMQMVrfeRnzPSPEBQwXb98qzYxKeDEFXMKZZ5Cl+8BgxtQw5MDJ5ORiidJmfa7mewV5fK9MrIGRRZl+GTgE9MTJGpq0rEaAHtA309BbSNSXGN6VWfhj/E/sHNaqGzgRqn3PDnwu/4kPl8QVpc="